app/controllers/api/v1/orders_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
       end
 
       def show
-        authorize Order
+        authorize(Order)
```

Comment:
@eclarizio This is missing the order object, it won't take a class since we are trying to check for a specific order
```
order = Order.find(params.require(:order_id))
authorize(order)
render :json => order
```